src/Request/ChangeItemQuantityRequest.php

```
@@ -31,7 +31,7 @@ public function __construct(Request $request)
     {
         $this->token = $request->attributes->get('token');
         $this->id = $request->attributes->get('id');
-        $this->quantity = $request->request->get('quantity');
+        $this->quantity = $request->request->has('quantity') ? $request->request->getInt('quantity') : null;
```

Quantity cannot be null, it's declared as `int`. `$request->request->getInt('quantity', 1)` instead?

---

`$request->request->getInt('quantity', 0)`? This should trigger validation error. Question is, what should be a default behaviour. 1 as a default would be more flexible for frontends...

---

If we allow not to pass quantity then it makes sense to use 1 as the default value. If we require quantity then we should throw an exception here if it does not exist.

---

The decision is up to you.

---

Let's add `1` as default. But we need to mention it in docs.

---

Can you adjust it, please?

---

sure, but it will take some time